index.html

```diff
@@ -65,686 +68,33 @@
-					dolore dolores ea voluptua takimata aliquyam dolore justo. Erat sed dolor sit amet elitr est eirmod.
-					Sadipscing aliquyam lorem est elitr elitr eos accusam gubergren. Et lorem amet sadipscing takimata diam
-					tempor. Dolore takimata tempor amet lorem. Tempor sed dolor magna eirmod labore. Dolore stet et ipsum aliquyam
-					erat dolor lorem. Amet sed duo voluptua lorem vero amet gubergren ipsum dolor, accusam est sanctus lorem
-					consetetur magna. Duo et ipsum no gubergren sadipscing justo et. Vero accusam dolor voluptua rebum et tempor
-					ipsum, dolor gubergren vero invidunt sit eos accusam consetetur magna amet, et diam tempor et et eirmod kasd,
-					clita lorem lorem stet consetetur rebum dolor diam amet, magna clita no et no clita rebum sed takimata sea,
-					duo lorem kasd sanctus no sanctus takimata ut et justo. Sea sit est amet rebum et at at lorem et, sed takimata
-					erat sed elitr, et gubergren justo accusam justo. Amet at takimata stet dolores lorem. Elitr rebum ea magna
-					sadipscing dolor. Magna eirmod gubergren dolor amet sit stet dolor erat sed. Nonumy sea et at dolor tempor
-					tempor. Dolor dolore lorem eirmod lorem aliquyam ipsum sadipscing dolor vero. Et ea sit ut amet. Sit amet et
-					sadipscing sit magna sit, kasd invidunt sadipscing dolores et no lorem. Justo et rebum eirmod rebum lorem,
-					diam takimata nonumy magna consetetur et, lorem consetetur.
-				</ts-panel>
-				<ts-button-group slot="footer">
-					<ts-button type="primary" onclick="toggleCover('my-cover')">Toggle cover</ts-button>
-					<ts-button onclick="toggleAside('my-aside')">Toggle aside</ts-button>
-				</ts-button-group>
-			</ts-aside>
-			<ts-cover id="my-cover"></ts-cover>
-		</ts-root>
-		<script>
-			function onOpen() {
-				console.log('The aside is opening');
-			}
-
-			function onOpened() {
-				console.log('The aside is opened');
-			}
-
-			function onClose() {
-				console.log('The aside is closing');
-			}
-
-			function onClosed() {
-				console.log('The aside is closed');
-			}
-			document.getElementById('my-aside').addEventListener('open', onOpen);
-			document.getElementById('my-aside').addEventListener('opened', onOpened);
-			document.getElementById('my-aside').addEventListener('close', onClose);
-			document.getElementById('my-aside').addEventListener('closed', onClosed);
-		</script>
+		<article>
+			<h3>Radio</h3>
+			<form action="/index.php">
+				<!--				<fieldset class="radio-button-container" role="radiogroup" aria-labelledby="group_label_1">-->
+				<ts-radio-group name="radio">
+					<ts-radio value="blue" label="Blue pill"></ts-radio>
+					<ts-radio value="yellow" label="Yellow pill"></ts-radio>
+					<ts-radio value="black" label="Black pill" disabled></ts-radio>
+					<ts-radio value="white" label="White pill" checked></ts-radio>
+					<ts-radio value="green" label="Green pill"></ts-radio>
+				</ts-radio-group>
+				<!--				</fieldset>-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a space at column 23. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AWhiteSpaceAroundCheck)